RFXtrx/__init__.py

```diff
@@ -303,6 +303,8 @@ def __init__(self, pkt):
         super(SensorEvent, self).__init__(device)
 
         self.values = {}
+        if isinstance(pkt, lowlevel.RfxMeter):
+            self.values['Value'] = pkt.value
```

Comment:
Ok, lets use counterValue. When you have fixed that, I will make a new release of the rfxtrx lib, and add it to Home Assistant.